pkgs/development/tools/fdroidserver/default.nix

```diff
@@ -22,21 +20,25 @@
 , qrcode
 , requests
 , ruamel-yaml
+, sdkmanager
+, setuptools
 , yamllint
 }:
 
 buildPythonApplication rec {
   pname = "fdroidserver";
-  version = "unstable-2023-10-23";
-  format = "setuptools";
+  version = "2.2.2";
```

Comment:
This is mostly a downgrade and breaks the format function. Please update to 2.3a1 instead. We shuold also update androguard to 4.1 maybe with an override.